buda/entities/signup.proto

```diff
@@ -18,11 +18,21 @@ message Signup {
 
     enum SignupClient {
         WEB = 0;
-        IOS = 1;
-        ANDROID = 2;
-        OTHER = 3;
+        PUBLISH = 1;
```

Comment:
Sorry for any confusion I might have caused on this, but I was originally thinking this would be the api client.  Maybe I am missing some context though, is Publish and Analyze their own api clients now?

Reply:
Ah, gotcha, ok thanks for clarifying! 😄 
Dan mentioned in https://github.com/bufferapp/buffer-js-buffermetrics/issues/27 that "We're only using client_id to track web signups, we'll need to add this to the API signups too to get all data for signup_client ([add to this method](https://github.com/bufferapp/buffer-web/blob/10ca6c634b90120738d479f4e250c30779454ca9/shared/models/user_model.php#L5572-L5594))
We can extract this into a separate library to use across both app and api codebases."
Does that maybe help?